### PR TITLE
Add support for t2a models

### PR DESCRIPTION
**Pull request by agapioscon**
TODO:
- Allow users to specify model

### CODE REVIEW

**Comment by pyamin1878** [comment]
audiocraft is awesome, great contribution!

File: aictl/cli/main.py [hunk 1]
```diff
@@ -353,6 +371,21 @@ def main():
     )
     t2v_parser.set_defaults(func=t2v)
 
+    t2a_parser = subparsers.add_parser("t2a", help="the text-to-audio subcommand")
```
Comment (trizko) [inline aictl/cli/main.py:383]: For the model, let's add the model argument here and in the help section specify the 4 options:
```python
    t2a_parser.add_argument(
        "-m", "--model", default="small", help="the MusicGen model to use (options: small, medium, large, melody)"
    )
```

File: aictl/cli/main.py [hunk 2]
```diff
@@ -161,6 +161,24 @@ def t2v(args):
...
+    model = MusicGen.get_pretrained("melody")
+
+    # Set the duration
+    model.set_generation_params(duration=int(args.duration))
+
+    # Set the description, for now one at a time
+    wav = model.generate([args.prompt])
+
+    # TODO set strategy to arguments
+    audio_write(args.output_path, wav[0].cpu(), model.sample_rate, strategy="loudness")
```
Comment (trizko) [inline aictl/cli/main.py]: It seems like we need to install `ffmpeg` in order to use this function. Would it be needed with the `scipy` method of saving the audio file? Example:
```python
import scipy

sampling_rate = model.config.audio_encoder.sampling_rate
scipy.io.wavfile.write("musicgen_out.wav", rate=sampling_rate, data=audio_values[0, 0].numpy())
```

**Review by trizko — COMMENTED** [review]
Thanks for this! I left a couple comments and I'm guessing we will also need to add the audiocraft dependency to `setup.py`.

File: aictl/cli/main.py (hunk 2 above)
Comment (agapioscon) [reply aictl/cli/main.py]: Correct me if I am wrong but ffmpeg is required for audiocraft. Is there a reason why we don't want ffmpeg?


From audiocraft git :  https://github.com/facebookresearch/audiocraft
**Note: Please make sure to have [ffmpeg](https://ffmpeg.org/download.html) installed when using newer version of torchaudio.'**

Comment (trizko) [reply aictl/cli/main.py]: Resolved this conversation because it looks like we don't need `ffmpeg` to run the `audio_write` function.

**Comment by trizko** [comment]
in the setup.py, we can add this block to the `extra_requires` section and allow users to opt out of the audiocraft dependencies:
```python
        'audio': [
            'audiocraft @ git+https://github.com/facebookresearch/audiocraft.git'
        ],
```

**Review by trizko — APPROVED** [review]
LGTM Thanks!

**Pull request merged**